iphone/Maps/Bookmarks/Categories/Sharing/Tags/TagsLayout.swift

```diff
@@ -0,0 +1,109 @@
+final class TagsLayout: UICollectionViewLayout {
+  enum Element: String {
+    case header
+    case cell
+  }
+
+  private var cache = [Element : [IndexPath : UICollectionViewLayoutAttributes]]()
```

Comment:
I don't like the idea of having a dictionary of dictionaries. Consider replacing it with two separate dictionaries